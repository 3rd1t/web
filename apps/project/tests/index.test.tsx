import React from "react"
import renderer from "react-test-renderer"
import Index from "../pages/index"

describe(" Index", () => {
  it("renders correctly", () => {
    const codeExamples = [
      {
        language: "python",
        snippet: "hello snippet",
      },
      {
        language: "r",
        snippet: "hello snippet",
      },
    ]

    const tree = renderer.create(<Index codeExamples={codeExamples} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
