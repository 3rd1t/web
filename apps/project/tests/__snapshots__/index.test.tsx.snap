// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Index renders correctly 1`] = `
Array [
  <section
    className="relative min-h-screen"
    id="index"
  >
    <div
      className="flex flex-col items-start px-4 pt-20 space-y-8 sm:pt-24 sm:px-6 md:pt-28 lg:pt-32 lg:px-8 xl:pt-40 xl:flex-row xl:space-y-0 xl:space-x-8"
    >
      <div
        className="flex flex-col items-center w-full"
      >
        <div
          className="space-y-4 text-left sm:text-center md:space-y-8 xl:space-y-12 lg:text-left"
        >
          <h2
            className="text-4xl font-extrabold leading-10 tracking-tight text-gray-900 xl:text-5xl sm:leading-none md:text-5xl"
          >
            <h1>
              Portfolio analytics insights for everyone
            </h1>
          </h2>
          <div
            className="text-gray-600 sm:text-lg sm:mx-auto md:text-xl lg:mx-0"
          >
            <p
              className="flex flex-col xl:flex-row"
            >
              Giving you access to the latest investment portfolio analytics methods in science.
            </p>
          </div>
          <div
            className="space-y-4 md:space-x-4 md:space-y-0"
          >
            <div
              className="sm:flex sm:justify-center lg:justify-start"
            >
              <div
                className="rounded-md shadow"
              >
                <a
                  className="border border-transparent rounded-sm md:py-4 md:text-lg md:px-10 flex items-center justify-center w-full px-8 py-3 text-base font-medium leading-6 transition duration-150 ease-in-out focus:outline-none text-white bg-gray-900 hover:bg-gray-800 hover:text-gray-100"
                  href="#"
                >
                  Get started
                </a>
              </div>
              <div
                className="mt-3 sm:mt-0 sm:ml-3"
              >
                <a
                  className="border border-transparent rounded-sm md:py-4 md:text-lg md:px-10 flex items-center justify-center w-full px-8 py-3 text-base font-medium leading-6 transition duration-150 ease-in-out focus:outline-none text-gray-700 bg-gray-100 hover:text-gray-600 hover:bg-white"
                  href="Log in"
                >
                  Log in
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="absolute inset-x-0 bottom-0 flex justify-center"
    >
      <div
        className="w-20 h-20 p-4 transform rotate-45 bg-gray-500 "
      >
        <div
          className="flex items-center justify-center transform -rotate-45"
        >
          <a
            href="#features"
          >
            Arrow down TODO:
          </a>
        </div>
      </div>
    </div>
  </section>,
  <section
    className="flex flex-col justify-center px-4 pt-20 mx-auto bg-white sm:px-6 lg:px-8"
    id="features"
  >
    <div
      className="md:text-center"
    >
      <h3
        className="mt-8 text-3xl font-bold leading-8 tracking-tight text-gray-900 sm:text-4xl sm:leading-10"
      >
        Insights we offer
      </h3>
      <p
        className="max-w-2xl mt-4 text-xl leading-7 text-gray-500 md:mx-auto"
      >
        Lorem ipsum dolor sit amet consect adipisicing elit. Possimus magnam voluptatum cupiditate veritatis in accusamus quisquam.
      </p>
    </div>
    <div
      className="flex flex-col px-5 py-24 mx-auto lg:flex-row lg:space-x-8"
    >
      <div
        className="relative flex pb-12 lg:items-center lg:w-1/4 lg:flex-col"
      >
        <div
          className="absolute inset-0 flex items-center justify-center sm:w-12 w-10 md:w-12 h-full lg:hidden"
        >
          <div
            className="w-1 h-full bg-gray-200 pointer-events-none"
          />
        </div>
        <div
          className="relative z-10 inline-flex items-center justify-center flex-shrink-0 w-10 h-10 p-1 text-gray-900 bg-gray-100 border-4 border-gray-900 rounded-full lg:bg-gray-900 lg:text-gray-100 lg:border-0 border-3 sm:w-12 sm:h-12 md:w-12 md:h-12 lg:rounded"
        >
          <svg
            fill="currentColor"
            stroke="currentColor"
            viewBox="0 0 20 20"
          >
            <path
              clipRule="evenodd"
              d="M16.707 5.293a1 1 0 010 1.414l-8 8a1 1 0 01-1.414 0l-4-4a1 1 0 011.414-1.414L8 12.586l7.293-7.293a1 1 0 011.414 0z"
              fillRule="evenodd"
            />
          </svg>
        </div>
        <div
          className="flex-grow pl-4 lg:pl-0 lg:mt-8"
        >
          <h2
            className="mb-1 text-lg font-medium leading-6 text-gray-900"
          >
            Independent
          </h2>
          <p
            className="text-base leading-6 text-gray-600"
          >
            Morbi ultricies molestie lacinia. Vivamus sagittis congue eleifend. Phasellus congue porttitor risus, a imperdiet enim tristique vel. Praesent libero dolor. 
          </p>
        </div>
      </div>
      <div
        className="relative flex pb-12 lg:items-center lg:w-1/4 lg:flex-col"
      >
        <div
          className="absolute inset-0 flex items-center justify-center sm:w-12 w-10 md:w-12 h-full lg:hidden"
        >
          <div
            className="w-1 h-full bg-gray-200 pointer-events-none"
          />
        </div>
        <div
          className="relative z-10 inline-flex items-center justify-center flex-shrink-0 w-10 h-10 p-1 text-gray-900 bg-gray-100 border-4 border-gray-900 rounded-full lg:bg-gray-900 lg:text-gray-100 lg:border-0 border-3 sm:w-12 sm:h-12 md:w-12 md:h-12 lg:rounded"
        >
          <svg
            fill="currentColor"
            stroke="currentColor"
            viewBox="0 0 20 20"
          >
            <path
              clipRule="evenodd"
              d="M16.707 5.293a1 1 0 010 1.414l-8 8a1 1 0 01-1.414 0l-4-4a1 1 0 011.414-1.414L8 12.586l7.293-7.293a1 1 0 011.414 0z"
              fillRule="evenodd"
            />
          </svg>
        </div>
        <div
          className="flex-grow pl-4 lg:pl-0 lg:mt-8"
        >
          <h2
            className="mb-1 text-lg font-medium leading-6 text-gray-900"
          >
            Fair pricing - start for free!
          </h2>
          <p
            className="text-base leading-6 text-gray-600"
          >
            Nam imperdiet metus quis sapien egestas sagittis. Cras venenatis, orci non tempus porttitor, quam lorem sodales mauris, in fermentum purus. 
          </p>
        </div>
      </div>
      <div
        className="relative flex pb-12 lg:items-center lg:w-1/4 lg:flex-col"
      >
        <div
          className="absolute inset-0 flex items-center justify-center sm:w-12 w-10 md:w-12 h-full lg:hidden"
        >
          <div
            className="w-1 h-full bg-gray-200 pointer-events-none"
          />
        </div>
        <div
          className="relative z-10 inline-flex items-center justify-center flex-shrink-0 w-10 h-10 p-1 text-gray-900 bg-gray-100 border-4 border-gray-900 rounded-full lg:bg-gray-900 lg:text-gray-100 lg:border-0 border-3 sm:w-12 sm:h-12 md:w-12 md:h-12 lg:rounded"
        >
          <svg
            fill="currentColor"
            stroke="currentColor"
            viewBox="0 0 20 20"
          >
            <path
              clipRule="evenodd"
              d="M16.707 5.293a1 1 0 010 1.414l-8 8a1 1 0 01-1.414 0l-4-4a1 1 0 011.414-1.414L8 12.586l7.293-7.293a1 1 0 011.414 0z"
              fillRule="evenodd"
            />
          </svg>
        </div>
        <div
          className="flex-grow pl-4 lg:pl-0 lg:mt-8"
        >
          <h2
            className="mb-1 text-lg font-medium leading-6 text-gray-900"
          >
            High data quality
          </h2>
          <p
            className="text-base leading-6 text-gray-600"
          >
            Nullam dapibus turpis lorem, at porta risus hendrerit nec. Maecenas eu ornare felis. Praesent fringilla libero magna, in tempus dui. 
          </p>
        </div>
      </div>
      <div
        className="relative flex pb-12 lg:items-center lg:w-1/4 lg:flex-col"
      >
        <div
          className="absolute inset-0 flex items-center justify-center sm:w-12 w-10 md:w-12 h-full hidden"
        >
          <div
            className="w-1 h-full bg-gray-200 pointer-events-none"
          />
        </div>
        <div
          className="relative z-10 inline-flex items-center justify-center flex-shrink-0 w-10 h-10 p-1 text-gray-900 bg-gray-100 border-4 border-gray-900 rounded-full lg:bg-gray-900 lg:text-gray-100 lg:border-0 border-3 sm:w-12 sm:h-12 md:w-12 md:h-12 lg:rounded"
        >
          <svg
            fill="currentColor"
            stroke="currentColor"
            viewBox="0 0 20 20"
          >
            <path
              clipRule="evenodd"
              d="M16.707 5.293a1 1 0 010 1.414l-8 8a1 1 0 01-1.414 0l-4-4a1 1 0 011.414-1.414L8 12.586l7.293-7.293a1 1 0 011.414 0z"
              fillRule="evenodd"
            />
          </svg>
        </div>
        <div
          className="flex-grow pl-4 lg:pl-0 lg:mt-8"
        >
          <h2
            className="mb-1 text-lg font-medium leading-6 text-gray-900"
          >
            Sophisticated analytics methods - simply illustrated
          </h2>
          <p
            className="text-base leading-6 text-gray-600"
          >
            Morbi at mauris et dolor rutrum molestie. In venenatis dictum tellus ut varius. Cras sed ante tincidunt, condimentum lacus eu.
          </p>
        </div>
      </div>
    </div>
  </section>,
  <section
    className="flex flex-col justify-center px-4 py-20 mx-auto bg-gray-100 sm:px-6 lg:px-8"
    id="team"
  >
    <div
      className="flex justify-center"
    >
      <h2
        className="text-3xl font-extrabold leading-8 tracking-tight text-gray-900 sm:text-4xl sm:leading-10"
      >
        Our Team
      </h2>
    </div>
    <div
      className="flex flex-col px-5 py-16 mx-auto"
    >
      <div
        className="mx-auto lg:w-4/6"
      >
        <div
          className="overflow-hidden rounded-sm"
        >
          <img
            alt="team photo"
            className="object-cover w-full"
            src="./team.jpg"
          />
        </div>
        <div
          className="flex flex-col mt-10 sm:flex-row"
        >
          <div
            className="text-center sm:w-1/3 sm:pr-8 sm:py-8"
          >
            <div
              className="inline-flex items-center justify-center w-20 h-20 bg-gray-900 rounded-full"
            >
              <svg
                className="p-3 text-gray-100 stroke-current"
                fill="none"
                viewBox="0 0 194 148"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M9 106.208L97.3883 17.8199L134.158 54.5894M185.07 41.8615L96.6814 130.25L59.9118 93.4803"
                  strokeWidth="25"
                />
              </svg>
            </div>
            <div
              className="flex flex-col items-center justify-center mt-4 space-y-2 text-center"
            >
              <h2
                className="text-lg font-medium leading-6 text-gray-900"
              >
                Perfolio
              </h2>
              <p
                className="text-base leading-6 text-left text-gray-600 sm:text-center"
              >
                We are all super impressed by the performance of our product owners from amos and would like to thank them for everything they have taught us.
              </p>
            </div>
          </div>
          <div
            className="pt-4 mt-4 border-t border-gray-300 sm:w-2/3 sm:pl-8 sm:py-8 sm:border-l sm:border-t-0 sm:mt-0"
          >
            <p
              className="text-lg leading-6 text-center text-gray-600 sm:text-left"
            >
              Mauris porta, turpis eu rhoncus accumsan, neque tortor feugiat purus, vel egestas tellus magna nec leo. Pellentesque ultrices posuere consequat. Quisque mollis erat eget aliquam mollis. Aenean convallis aliquet augue venenatis posuere. Donec pretium ante nunc, eu tempus diam dictum quis. Etiam nunc magna, tincidunt et porttitor at, volutpat rutrum felis. Nullam condimentum consectetur urna non pharetra. Vivamus enim massa, porttitor a lobortis dignissim, iaculis id arcu. Aliquam feugiat nisl sed blandit pulvinar. Sed bibendum rhoncus erat, et feugiat dolor ullamcorper non. Sed interdum et justo non aliquet. 
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>,
  <section
    className="flex flex-col justify-center px-4 py-20 mx-auto bg-white sm:px-6 lg:px-8"
    id="subscribe"
  >
    <div
      className="container flex flex-col justify-around mx-auto lg:flex-row"
    >
      <div>
        <h2
          className="text-2xl font-semibold leading-8 text-gray-900 font-display sm:text-3xl sm:leading-9"
        >
          Curious for more?
        </h2>
        <p
          className="max-w-2xl mt-2 text-base leading-6"
        >
          <span
            className="text-gray-600"
          >
            Follow us on our roadmap and on 
          </span>
          <a
            href="https://github.com/perfolio"
            target="blank"
          >
            github
          </a>
          .
        </p>
      </div>
      <form
        className="mt-6"
      >
        <div
          className="sm:flex"
        >
          <input
            className="block w-full px-4 py-3 text-base leading-6 placeholder-gray-600 bg-white border border-gray-300 rounded shadow-none sm:max-w-xs focus:outline-none focus:border-gray-700"
            placeholder="Enter your email"
            type="email"
          />
          <button
            className="relative block w-full px-6 py-3 mt-4 text-base font-semibold leading-6 text-white transition duration-150 ease-in-out border border-transparent rounded shadow-md bg-carbon-900 sm:mt-0 sm:h-auto sm:ml-4 sm:w-auto hover:bg-carbon-800 focus:outline-none focus:bg-gray-800 hover:bg-gray-600"
          >
            Subscribe
          </button>
        </div>
      </form>
    </div>
  </section>,
]
`;
